pkgs/development/python-modules/wolframclient/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyzmq
+, numpy
+, pytz
+, requests
+, aiohttp
+, oauthlib
+}:
+
+buildPythonPackage rec {
+  pname = "wolframclient";
+  version = "1.1.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0vhgmr9i6qa98qv8sn7ph35m2gayjyif6d2q0sjbfdgihsh0ly8f";
+  };
+
+  buildInputs = [
```

python packages should be in propagatedBuildInputs
```suggestion
  propagatedBuildInputs = [
```

---

Right...

---

Thanks.